### PR TITLE
Improve webhooks data

### PR DESCRIPTION
**Pull request by bameda**
Improve webhook data: 
- add permalinks
- owner, status, type, priority, severity, user_story, milestone, project are objects
- add the owner to every notification ('by' field)
- add the date of the notification ('date' field)
- show human diffs in changes and remove unnecessary data
  
Related to:  https://github.com/taigaio/taiga-doc/pull/122/files

Fix:
- https://tree.taiga.io/project/taiga/issue/3688 
- https://tree.taiga.io/project/taiga/issue/3755

Preview of the new doc: https://github.com/taigaio/taiga-doc/blob/updates/2.1/webhooks.adoc

### CODE REVIEW

**Comment by khasinski** [comment]
:+1:

**Comment by bameda** [comment]
:horse_racing:

**Comment by justi** [comment]
👍

**Comment by superalex** [comment]
@bameda I think a lot of external integrations with Taiga will love this PR :dancer: 

Just a question: does it make sense including this enhancement request too?: https://tree.taiga.io/project/taiga/issue/3688 we could change project to work like the user field, a container of id, slug and title for example.

**Comment by bameda** [comment]
@superalex https://tree.taiga.io/project/taiga/issue/3758 is talking about the API and not about webhooks.

**Comment by bameda** [comment]
@superalex now is ready to merge and you can see the doc about this here: https://github.com/taigaio/taiga-doc/blob/updates/2.1/webhooks.adoc

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,12 @@
 ## 2.1.0 ??? (unreleased)
 
 ### Features
-- ...
+- Webhooks: Improve webhook data: 
+    - add permalinks
+    - owner, status, type, priority, severity, user_story, milestone, project are objects
+    - add the owner to every notuification ('by' field)
```
Comment (superalex) [inline CHANGELOG.md]: notuification -> notification

**Comment by superalex** [comment]
There are errors when:
- Updating a user story status estimation
- Updating a user story status custom field
- Adding attachments to a user story
- Removing attachments from a user story
....

I think we could launch the e2e tests with a sample_data of projects using webhooks.

**Comment by bameda** [comment]
@superalex I created some test and now is fine for me.

**Comment by bameda** [comment]
@superalex "now is when" :-p

**Comment by superalex** [comment]
Great! :dancers:

**Pull request merged**